auth0/src/test/java/com/auth0/android/management/UsersAPIClientTest.java

```diff
@@ -48,7 +48,6 @@
 import com.google.gson.reflect.TypeToken;
 import com.squareup.okhttp.mockwebserver.RecordedRequest;
 
-import org.apache.tools.ant.filters.StringInputStream;
```

Comment:
wrong import being used. This class doesn't exist in this project, so the usage was replaced with a `ByteArrayInputStream` in 4 files.